### PR TITLE
New package: Netsurf-fb

### PR DESCRIPTION
**Pull request by Spryc**

### CODE REVIEW

**Comment by ericonr** [comment]
You can run xlint manually from the xtools package. That way you can verify the template locally. The PR should be in the form of a single commit as well, per the instructions in `CONTRIBUTING.md`.

**Comment by ahesford** [comment]
Please try building this package locally before pushing commits, or add `[ci skip]` to your commit messages while you work to avoid flooding the Travis queue with builds that are destined to fail. Once you have a successful build on your local machine, you can either push your final work or amend the commit message to remove the `[ci skip]`.

**Comment by Spryc** [comment]
Okay, will do that 
Thanks

File: srcpkgs/netsurf-fb/template [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+pkgname=netsurf-fb
+wrksrc=netsurf-3.9-src
+version=3.9
+revision=1
+hostmakedepends="pkg-config gperf flex nsgenbind gdk-pixbuf-devel
+ tar perl-HTML-Parser glib-devel xxd"
+makedepends="
+ libidn-devel libmng-devel libxml2-devel libcurl-devel lcms-devel libwebp-devel
+ libcss-devel libnsbmp-devel libnsgif-devel libutf8proc-devel
+ libnsutils-devel libdom-devel gtk+-devel"
```
Comment (ahesford) [inline srcpkgs/netsurf-fb/template:11]: Is GTK actually required for a framebuffer build?

**Comment by ahesford** [comment]
Also, if you're going to pushing a lot, you should mark the PR a draft to indicate that it is not yet ready.

**Comment by Hoshpak** [comment]
This PR should be a build option instead of a new template.

**Pull request closed without merging**